### DISCUSSION TITLE
Auto resolve conversation after reply

### DISCUSSION THREAD

**Discussion by Christilut** [chatwoot/chatwoot#3912] [Feature Requests]
When answering a chat, I would like the conversation to immediately resolve because I always resolve it by hand after that anyway.

It happens a lot that website visitors ask something but never reply. After I send my reply, I don't need that conversation anymore until they reply. So automatically resolving it makes a lot of sense here.

**Comment by sojan-official** [comment]
@Christilut  Could be fixed by our automation rules feature which will be rolled out soon 
ref: https://github.com/chatwoot/chatwoot/issues/3195

you should be able to configure a rule that will resolve conversation on agent reply. 

----

you can also try our keyboard shortcuts.  which could be a  faster workflow. 
<img width="905" alt="Screenshot 2022-02-07 at 7 06 18 PM" src="https://user-images.githubusercontent.com/73185/152910482-7c25e0c6-e43a-47aa-9a71-e7ca8ddec83f.png">

----

 you could also utilize the auto-resolve configuration in account settings which will make the system resolve inactive conversations. the minimum configuration is 1 day.

**Reply by Christilut** [reply]
Thanks. Automation rule could work very well and keyboard shortcut is a good tip, I'll definitely use those.
